### ISSUE TITLE
Issues reported from WEKA team

### ISSUE THREAD

**Issue by berndbischl** [mlr-org/mlr#607]
Hi Bernd,

I’ve just finished upgrading the RPlugin for WEKA to use the classification and regression schemes from mlr 2.7. It’s very cool to have xgboost in there, which seems to be one of the current go-to methods.

When going through all the schemes manually in WEKA’s Explorer GUI, I noticed a couple of potential issues:

* classif.LiblineaRL2SVC seems to actually invoke LibLINEAR for L2R_LR:

---

Learner classif.LiblineaRL2SVC from package LiblineaR
Type: classif
Name: L2-Regularized L2-Loss Support Vector Classification; Short name: liblinl2svc
Class: classif.LiblineaRL2SVC
Properties: twoclass,multiclass,numerics,class.weights
Predict-Type: response
Hyperparameters: 

$TypeDetail
[1] "L2-regularized logistic regression primal (L2R_LR)”

---

* regr.penalized.lasso and regr.penalized.ridge appear to return exactly the same results (with default settings)

* regr.randomForestSRC does not work a prediction time: "Error in checkPredictLearnerOutput(.learner, .model, p) : predictLearner for regr.randomForestSRC has returned a class array instead of a numeric!”

**Comment by berndbischl** [comment]
For penalized learners I have added a separate issue here: #612

**Comment by florianfendt** [comment]
I put up a PR for the LiblineaR learners!

**Comment by berndbischl** [comment]
The liblinear thing is fixed.

the penalized issue is not really a issue. the default for lasso and ridge are both lambda = 0, so both fit a linear model. so the must be the same.

**Comment by berndbischl** [comment]
The randdomForestSRC problem was due to an API change in that package.
I took care of this even before you emailed me.

https://github.com/mlr-org/mlr/commit/2f14ca0d66cbed182752f0f006510f881319304a

**Comment by berndbischl** [comment]
So thanks again closing.
